guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_prerequisites.adoc

```diff
@@ -19,6 +19,10 @@ Cloning is the recommended method because that prevents them being overwritten i
 To confirm if a template has been edited, you can view its *History* before you upgrade or view the changes in the audit log after an upgrade.
 In the {ProjectWebUI}, navigate to *Monitor* > *Audits* and search for the template to see a record of changes made.
 If you use the export method, restore your changes by comparing the exported template and the default template, manually applying your changes.
+ifdef::satellite[]
+* If you previously upgraded {Project} from an earlier version, or if the `/var/lib/pgsql` directory has any content, ensure it is empty before proceeding.
```

Comment:
`or` feels odd to me. Isn't it `and`? Or rather: if `/var/lib/pgsql` has any content (as is the case when upgrading from older versions), empty it.